boa/src/builtins/number/mod.rs

```diff
@@ -417,7 +417,23 @@ impl Number {
         make_builtin_fn(Self::to_string, "toString", &prototype, 1);
         make_builtin_fn(Self::value_of, "valueOf", &prototype, 0);
 
-        make_constructor_fn(Self::make_number, global, prototype)
+        let number = make_constructor_fn(Self::make_number, global, prototype);
+
+        // Constants from:
+        // Draft ECMA-262 / May 22, 2020 ECMAScript® 2021 Language Specification
+        // Section 20.1.2 Properties of the Number Constructor
+        number.set_field("EPSILON", Value::from(std::f64::EPSILON));
+        number.set_field("MAX_SAFE_INTEGER", Value::from(9_007_199_254_740_991_f64));
+        number.set_field("MIN_SAFE_INTEGER", Value::from(-9_007_199_254_740_991_f64));
+        number.set_field("MAX_VALUE", Value::from(std::f64::MAX));
+        number.set_field("MIN_VALUE", Value::from(std::f64::MIN));
+        number.set_field("NEGATIVE_INFINITY", Value::from(f64::NEG_INFINITY));
+        number.set_field("POSITIVE_INFINITY", Value::from(f64::INFINITY));
+
+        // TODO: Currently parsing/lexing Number.NaN is not supported due to the inclusion of the lower case 'a'.
+        // number.set_field("NaN", Value::from(f64::NAN));
```

Comment:
I would still add this here, so that when we fix the lexer this will just work.

Reply:
> I would still add this here

Yep. We could add it! That's probably the best option.

Reply:
Will add it back :)

Reply:
I created #421 related to this.